cfme/configure/access_control.py

```diff
@@ -1294,7 +1301,8 @@ def is_displayed(self):
         return (
             self.form.accordions.accesscontrol.is_opened and
             any([self.form.title.text == 'Editing Tenant "{}"'.format(self.context['object'].name),
-                self.form.title.text == 'Editing Project "{}"'.format(self.context['object'].name)])
+                 self.form.title.text == 'Editing Project "{}"'.format(
```

Comment:
**Required**

I see `any()` was used here, before, but same as above I would like to see a unique identification of the view based on the contextual object.